readme.md

```diff
@@ -22,271 +22,13 @@ Support for multiple Jira instances is considered, but not yet supported.
-
-```
-https://community.mattermost.com/plugins/jira/webhook?secret=5JlVk56KPxX629ujeU3MOuxaiwsPzLwh&team=contributors&channel=town-square&updated_comments=1
-```
-
-This sends all comment notifications to a Mattermost channel, including public and private comments, so be cautious of which channel you send these notifications to.
-
-## 5. Troubleshooting
-
-**Note:** If you experience problems with Jira-related user interactions in Mattermost such as creating issues, disable these features by setting **Allow users to connect their Mattermost accounts to Jira** to false in **System Console > Plugins > Jira**. This setting does not affect Jira webhook notifications. After changing this setting to false, disable, then re-enable this plugin in **System Console > Plugins > Plugin Management** to reset the plugin state for all users.
-
-Sometimes the plugin may crash unexpectedly and you may notice a response in red text below the chat window displaying ` slash command with trigger of  '/(name)' not found,`. If you check your log file, look for messages that refer to "plugins" and "health check fail", "ExecuteCommand" etc.
-
-If you encounter these types of issues you can set `LogSettings.FileLevel` to `DEBUG` in your `config.json` settings. This will enable debug logging and give more verbose error events in the system log. Then try re-enabling the plugin in the Mattermost System Console. These log results may be requested by others in the forum or by our support team.
-
-**Note:** If you have a site with high volumes of activity, this setting can cause log files to expand substantially and may adversely impact the server performance. Keep an eye on your server logs, or only enable in development environments.
-
-### Jira/Mattermost User Connections
-
-Connecting an account between Mattermost and Jira is a key part of the installation process and requires the end-user to authenticate with Jira and allow access to their Jira account. All `create`, `view`, `assign`, and `transition` operations are done using the logged in user's Jira access token.
-
-- You must be signed into Mattermost on the same browser you are using to sign into Jira during `connect`.
-- The domain end users sign into Mattermost with on that browser must match the SiteURL in `config.json`.
+This plugin contains both a server and web app portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) for more information about developing and extending plugins.
 
-## 6. Help
+## Support
 
-- For Mattermost customers - please open a support case.
-- For Questions, Suggestions, and Help - please find us on our forum at https://forum.mattermost.org/c/plugins.
+- For Mattermost customers - Please open a support case.
+- For Questions, Suggestions, and Help - Please find us on our forum at https://forum.mattermost.org/c/plugins.
```

Comment:
Should we point people towards https://github.com/mattermost/mattermost-plugin-jira/issues and the Jira plugin channel instead? The integrations team barely uses the forum.

Reply:
The idea of this ticket is to add all people signed in into MSTeams Calling that are on the channel to the meeting.

Reply:
@justinegeffen What do you think about this?

Reply:
I missed this comment but I agree and have updated the text accordingly. :)